notification/slack.py

```diff
@@ -24,7 +25,7 @@
 description:
     - The M(slack) module sends notifications to U(http://slack.com) via the Incoming WebHook integration
 version_added: 1.6
-author: "Ramon de la Fuente (@ramondelafuente)"
+author: Ramon de la Fuente <ramon@delafuente.nl>, Stefan Berggren <nsg@nsg.cc>
```

Comment:
Is this appropriate? Should I remove my name? Is this reserved to the module maintainer or something else maybe? Same question for [L4](https://github.com/nsg/ansible-modules-extras/blob/full-attachments-support/notification/slack.py#L4)

I noticed that there are plenty of people that haven’t added their names.